### PR TITLE
VAGOV-885 Add script to download STG DB and sync into Lando.

### PR DESCRIPTION
**Pull request by ElijahLynn**
This is an interim solution until we have a systemd timer (cron)
automate backups to a central location where we can sanitize and pull
from.

### CODE REVIEW

File: scripts/sync-db.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+# Exit immediately if a command fails with a non-zero status code
+set -e
+
+# Setup
+source sync-db-var-config
```
Comment (ElijahLynn) [inline scripts/sync-db.sh]: Need to check if file exists and return error message if not.

**Comment by ElijahLynn** [comment]
Interesting build failure though
> Thin Pool has 10077 free data blocks which is less than minimum required 10116 free data blocks. Create more free space in thin pool or use dm.min_free_space option to change behavior

**Comment by ElijahLynn** [comment]
Demo recording of this here > https://zoom.us/recording/play/wBeI7LxmmzXHAYekEYPw9sBS-u2aVUeDSi02qc2HCvzPvJtMkZg_GxGwmx0Q4rxN?continueMode=true

File: scripts/sync-db.sh (hunk 1 above)
Comment (ElijahLynn) [reply scripts/sync-db.sh]: Actually, the preceding `set -e` takes care of this well enough.

**Comment by ElijahLynn** [comment]
@mouncifb K, ready for final review.

**Comment by mbenziane** [comment]
Created an environment test-vagovcms#pr-56<br><br> [button_visit-site](http://pr-56.va.agile6.com)

**Pull request merged**